libcentrifugo/metrics.go

```diff
@@ -109,6 +112,24 @@ type metricsRegistry struct {
 	mu sync.Mutex
 }
 
+func newMetricsHistogramRegistry() *hdrhistogram.HDRHistogramRegistry {
+	quantiles := []float64{50, 90, 99, 99.99}
+	var minValue int64 = 1        // as we record latencies in microseconds, min resolution 1mks
+	var maxValue int64 = 10000000 // as we record latencies in microseconds, max resolution 10s
```

Comment:
I suggest larger max value - a thing over 10 seconds is bad but if you have heavily loaded server for a few mins and all you can see is that 50% of requests took more than 10s it's hard to know if that's a bit bad (they all took 11 seconds for the peak) or awful (they all hung and took 5 mins before client timed out). HDR histograms are pretty space efficient so going up to 10 mins should be negligible. if space is a big deal then you can reduce low resolution - nothing is likely t be faster than a few hundred microseconds so you could start at 10 or 100.